spec-examples/src/aes_gcm/aes.rs

```diff
@@ -184,29 +168,30 @@ fn key_expansion_word(w0: Word, w1: Word, i: usize, nk: usize, nr: usize) -> Wor
     k
 }
 
-fn key_expansion_aes128(key: Key128, nk: usize, nr: usize) -> Bytes176 {
-    let mut key_ex = Bytes176::new().update_start(&key);
-    let mut i: usize;
-    for j in 0..40 {
-        i = j + 4;
-        let word = key_expansion_word(
-            Word::from_sub(&key_ex, 4 * i - 16, 4),
-            Word::from_sub(&key_ex, nk * i - 4, 4),
-            i,
-            nk,
-            nr,
-        );
-        key_ex = key_ex.update(4 * i, &word);
-    }
-    key_ex
+#[derive(Clone, Copy)]
+pub enum AesVariant {
+    Aes128,
+    Aes256
 }
-fn key_expansion_aes256(key: Key256, nk: usize, nr: usize) -> Bytes240 {
-    let mut key_ex = Bytes240::new().update_start(&key);
+
+fn key_expansion_aes(key: &ByteSeq, nk: usize, nr: usize, alg: AesVariant) -> ByteSeq {
+    let mut key_ex = ByteSeq::new(match alg {
+        AesVariant::Aes128 => 176,
+        AesVariant::Aes256 => 240
+    }).update_start(key);
```

Comment:
For readability maybe move the `update_start(key)` a line below?

Reply:
Done :)